<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>global_style_form</class>
 <widget class="QWidget" name="global_style_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>322</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Knot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="knot_width_spinner">
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorWidget" name="color">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Point Style</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="point_combo">
        <item>
         <property name="text">
          <string>Bevel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Miter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Round</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Outline</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="outline_width_spinner">
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>128.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorWidget" name="outline_color">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Pattern</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="outline_pattern_combo">
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>4</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No Line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash Dot</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QLineEdit</extends>
   <header>color_widget.hpp</header>
   <slots>
    <signal>color_changed(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>outline_color</sender>
   <signal>color_changed(QColor)</signal>
   <receiver>global_style_form</receiver>
   <slot>change_pen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outline_pattern_combo</sender>
   <signal>activated(int)</signal>
   <receiver>global_style_form</receiver>
   <slot>change_pen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outline_width_spinner</sender>
   <signal>valueChanged(double)</signal>
   <receiver>global_style_form</receiver>
   <slot>change_pen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>point_combo</sender>
   <signal>activated(int)</signal>
   <receiver>global_style_form</receiver>
   <slot>change_join_style(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>knot_width_changed(double)</signal>
  <slot>change_pen()</slot>
  <slot>change_join_style(int)</slot>
 </slots>
</ui>
